src/app/organizations/organization-details/manage-projects/manage-single-project/manage-single-project.component.html

```diff
@@ -41,7 +41,7 @@ <h4>{{states.project.name}}</h4>
                     <div class="tabs">
                         <tabset>
                             <tab heading="Details" (selectTab)="detailsShown=true">
-                                <ng-container *ngIf="(updateForm$ | async) as updateForm">
```

Comment:
You already have the subscription at the beginning of the file so you can share it throughout the template.